src/core/Note.cpp

```diff
@@ -114,7 +114,7 @@ void Note::setPos( const TimePos & pos )
 
 void Note::setKey( const int key )
 {
-	const int k = qBound( 0, key, NumKeys - 1 );
```

Comment:
This upper bound is different to the one in the constructor, which uses `NumKeys`. Out of scope here, of course, but probably should be investigated.